tools/integration_tests/implicitdir/implicitdir_test.go

```diff
@@ -166,35 +77,40 @@ func executeTest(m *testing.M) (successCode int) {
 	// Creating a temporary directory to store files
 	// to be used for testing.
 	var err error
-	tmpDir, err = os.MkdirTemp(mntDir, "tmpDir")
+	tmpDir, err := os.MkdirTemp(setup.MntDir(), "tmpDir")
 	if err != nil {
-		logAndExit(fmt.Sprintf("Mkdir at %q: %v", mntDir, err))
+		setup.LogAndExit(fmt.Sprintf("Mkdir at %q: %v", setup.MntDir(), err))
 	}
-
+	setup.SetTmpDir(tmpDir)
 	successCode = m.Run()
 
-	os.RemoveAll(mntDir)
+	os.RemoveAll(setup.MntDir())
 
 	return successCode
 }
 
-func executeTestForFlags(flags []string, m *testing.M) (successCode int) {
+func executeTestForFlags(flags [][]string, m *testing.M) (successCode int) {
```

Comment:
Before we could pass only one flag at a time
Eg.
1. --enable-storage-client-libaray 
Here I converted flags in 2D to pass multiple flags in one tests.
E.g. = 
1. --enable-storage-client-libaray , --implicit-dirs, --o=ro